### PR TITLE
improving lending docs for integrators

### PR DESCRIPTION
**Pull request by dzmitry-lahoda**
Signed-off-by: Dzmitry Lahoda <dzmitry@lahoda.pro>

### CODE REVIEW

File: frame/lending/README.md [hunk 1]
```diff
@@ -2,44 +2,26 @@
...
-
-On block initialization. If can withdraw, than withdraws. If can deposit back, deposits maximal part available. If must liquidate, than deposit back all.
-
-### On borrow
-
-If must liquidate or not enough funds(we got maximal amount during initialization), borrow fails.
+Lending is executed after Vault. On block initialization. If can withdraw from Vault, than withdraws. If can deposit back, deposits maximal part available. If must liquidate, than deposit back all.
 
-### On repay
+On borrow, if must liquidate or not enough funds(we got maximal amount during initialization), borrow fails.
```
Comment (hussein-aitlahcen) [inline frame/lending/README.md:25]: `Borrowing is not allowed if we must liquidate (the vault is expecting the strategy to be liquidated) or if we don't have enough funds.`

File: frame/lending/README.md [hunk 2]
```diff
@@ -2,44 +2,26 @@
...
 
-Repay, do nothing to vault. It actually allows to use some deposit asset by other transactions in same block to borrow (event if we should deposit back into vault).
+On repay, do nothing to vault. It actually allows to use some deposit asset by other transactions in same block to borrow (event if we should deposit back into vault).
```
Comment (hussein-aitlahcen) [inline frame/lending/README.md:27]: `When repaying, we do not transfer the amount back to the vault. This is done within the pallet hooks. It actually allow...`

File: frame/liquidations/README.md [hunk 3]
```diff
@@ -1,13 +1,12 @@
...
-Used to liquidate collateral from `lending` protocol.  Need to liquidate because of bad collateral factor. It should be sold on exchange as soon as possible.
+Liquidation uses DEX and auctions for swaps. 
 
-- off-chain bots monitor `lending` markets for liquidation opportunities of borrow pairs.
+Uses different engines depending on available liquidity and configuration. 
 
-- uses different engines depending on available liquidity
-
-- default engine is `dutch-auction`
+Each liqudaiton strategy parameters can be changed by owner of strategy.
```
Comment (hussein-aitlahcen) [inline frame/liquidations/README.md:7]: `by the strategy's owner`

**Review by hussein-aitlahcen — APPROVED** [review]

File: frame/lending/README.md [hunk 4]
```diff
@@ -2,44 +2,26 @@
...
+Market = Isolated Currency Pair  + Configuration. 
 
-Vault = Vault provides and recalls liquidity into/from Lending via protocol.
+Oracle = only Pairs with Prices are allowed.
 
-Lending uses Vault for operation to withdraw assets and deposit back.
+Vault = Vault provides and recalls liquidity into/from Lending via protocol. User must put amount of Borrow into Vault for Market creation. Lenders put borrow asset into Vault. Borrowers put collateral into Vault, get xCollateral, and use it to get borrow for collateral. Borrower pays rent for his position.
 
-Liquidation uses DEX for swaps.
+OCW(or anybody) watches for under collateralized Poisitons and sends them to Liquidations. Liquidator is  rewarded with rent payed by borrower.
```
Comment (andor0) [inline frame/lending/README.md:17]: 2 spaces between `is`  and `rewarded`

**Review by andor0 — APPROVED** [review]

File: frame/lending/README.md (hunk 2 above)
Comment (dzmitry-lahoda) [reply frame/lending/README.md:27]: fixed in lending to dali branch

File: frame/liquidations/README.md (hunk 3 above)
Comment (dzmitry-lahoda) [reply frame/liquidations/README.md:7]: fixed in lending to dali branch

File: frame/lending/README.md (hunk 4 above)
Comment (dzmitry-lahoda) [reply frame/lending/README.md:17]: fixed in lending to dali branch

**Pull request merged**